src/pages/schedule.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Layout from '../components/layout';
+import SEO from '../components/seo';
+import Schedule from '../components/SchedulePage/Schedule';
+
+
+const SchedulePage = () => {
+	return <Layout>
+		<SEO title='Schedule' />
```

Comment:
Optional:

```suggestion
		<SEO title='Workshop schedule' />
```

Reply:
Will there be a separate overarching schedule eventually? Or will we include things like the opening ceremony, lunch, etc. on this page?